Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -134,7 +84,102 @@ struct ComposerActionBar: View {
             alignment: .top
         )
     }
-    
+
+    var defaultMenu: some View {
+        HStack(spacing: 0) {
+            if !showPreview {
+                attachMediaView
+                expirationTimeView
+            }
+            Spacer()
+            previewToggleView
+        }
+    }
+
+    /// Attach Media View
+    ///
+    /// Used by ``ComposerActionBar/defaultMenu``.
+    private var attachMediaView: some View {
+        ImagePickerButton(cameraDevice: .rear, mediaTypes: [.image, .movie]) { imageURL in
+            Task {
+                await uploadImage(at: imageURL)
+            }
+        } label: {
+            Image.attachMediaButton
+                .foregroundColor(.secondaryTxt)
+                .frame(minWidth: 44, minHeight: 44)
+        }
+        .padding(.leading, 8)
+        .accessibilityLabel(Text(.localizable.attachMedia))
+    }
+
+    /// Expiration Time
+    private var expirationTimeView: some View {
+        Group {
+            if let expirationTime, let option = ExpirationTimeOption(rawValue: expirationTime) {
+                ExpirationTimeButton(
+                    model: option,
+                    showClearButton: true,
+                    isSelected: Binding(get: {
+                        self.expirationTime == option.timeInterval
+                    }, set: {
+                        self.expirationTime = $0 ? option.timeInterval : nil
+                    })
+                )
+                .accessibilityLabel(Text(.localizable.expirationDate))
+                .padding(12)
+            } else {
+                Button {
+                    subMenu = .expirationDate
+                } label: {
+                    Image.disappearingMessages
+                        .foregroundColor(.secondaryTxt)
+                        .frame(minWidth: 44, minHeight: 44)
+                }
+            }
+        }
+    }
+
+    /// Preview Toggle
+    private var previewToggleView: some View {
```

Comment:
For this, I'd suggest shortening it since we already know a `Toggle` is a `View` (and the type makes it clear).
```suggestion
    private var previewToggle: some View {
```